lib/js/admin-geo.js

```diff
@@ -13,22 +13,26 @@
     var marker;
     var currentPosition;
     
-    window.setupAdminGeoMap = function(rscLat, rscLng, elementId, rscZoomLevel) {
+    window.setupAdminGeoMap = function(rscLat, rscLng, elementId, rscZoomLevel, defaultLat, defaultLng, defaultZoom) {
 
         rscLat = parseFloat(rscLat);
         rscLng = parseFloat(rscLng);
         rscZoomLevel = parseInt(rscZoomLevel, 10);
+
+        defaultLat = parseFloat(defaultLat);
+        defautlLng = parseFloat(defautlLng);
```

Comment:
Wrong name: `defautlLng` instead of `defaultLng` (also visible in the JS console).